src/java/org/apache/cassandra/locator/AbstractReplicationStrategy.java

```diff
@@ -284,6 +286,14 @@ public RangesAtEndpoint getAddressReplicas(InetAddressAndPort endpoint)
         return getAddressReplicas(tokenMetadata.cloneOnlyTokenMap(), endpoint);
     }
 
+    /**
+     * Return the number of token-owning nodes.
+     */
+    protected int getSizeOfRingMemebers()
```

Comment:
Seems this is only used by EverywhereStrategy, not sure it needs to be in AbstractReplicationStrategy?

Reply:
I added this to avoid keeping a reference to `TokenMetadata` in EverywhereStrategy.